### ISSUE TITLE
[help|feature] add an option to set not use ANSI escape sequences for search print?

### ISSUE THREAD

**Issue by roachsinai** [cheat/cheat#607]
## Problem

I define a shell function:

```
ces(){$(cheat -s "$@" | grep -v '^\s\+' | cut -d ':' -f 1 | fzf --reverse --inline-info --tac --cycle +s -x --height 35%) | xargs -r -0 -o -t cheat -e}
```

which use fzf to fuzzy find cheatsheet's name which include the search keyword.

But after the upgrade to cheat 4.2.0. , the fzf entry are surrounded by ANSI escape chars like: `  [4mwrite.title[0m [2m(personal)[0m`.

I read the release log found you made a update for search format.

**Describe the solution you'd like**

Could you please add an option to enable to suppress these escape chars?

---

Thanks in advance!

**Comment by chrisallenlane** [comment]
Hi, @roachsinai 

Would you please upload a a screenshot of the malformed text? It will help me to understand where the problem is.

Thanks.

**Comment by roachsinai** [comment]
![image](https://user-images.githubusercontent.com/9500049/101282940-cc1dbd80-3812-11eb-8913-03596364fff6.png)

😃

**Comment by chrisallenlane** [comment]
This should be resolved as of the `4.2.7` release:

https://github.com/cheat/cheat/releases/tag/4.2.7

Please let me know if this does/doesn't solve the problem for you, and feel free to re-open this ticket if necessary. Thanks.
